### PR TITLE
Story/033 landing page

### PR DESCRIPTION
**Pull request by jessicamcmillen**

### CODE REVIEW

**Review by Conormc98 — APPROVED** [review]

**Review by JamesMatchett — APPROVED** [review]
Large diff due to different formatter settings but otherwise fine, happy to approve as have tested it manually

File: ExpressAPI/db.js [hunk 1]
```diff
@@ -135,6 +133,21 @@ exports.dataGenerateUser = function(data, callback){
...
+exports.dataGenerateUser = function(data, callback){
+    db.query('INSERT INTO userData SET ?', data,
+    function(error, results, fields){
+        if(error)
+        {
+            logger.error(error);
+            callback(null, error)
+        }
+        logger.debug("Generating user "+ data.name)
+        console.log("results are")
```
Comment (micalexis) [inline ExpressAPI/db.js]: get rid

File: kjourney/src/app/main/main.component.html [hunk 2]
```diff
@@ -1,35 +1,106 @@
...
+
+		<div class="data">
+			<div class="whole">
+				<div class="miniLeft">
+					<h2>Capability:<li *ngFor="let userInfo of userInfo">{{ userInfo.capability }}</h2>
+					</div>
+					<div class="miniRight">
+						<h2>
+							<i id="click" class="material-icons" style="font-size: 30px; float:center;">info</i>
+						</h2>
```
Comment (micalexis) [inline kjourney/src/app/main/main.component.html:47]: put styles in css files

File: kjourney/src/app/main/main.component.ts [hunk 3]
```diff
@@ -1,25 +1,50 @@
...
   constructor(data: DataService) {
-    this.data = data
+    this.data = data;
   }
 
   async ngOnInit() {
-    this.data.getRoles().subscribe((c => {
-      this.roles = c;
-    }));
+    this.data.getUserInfo().subscribe((r:any) => {
```
Comment (micalexis) [inline kjourney/src/app/main/main.component.ts]: r? change to name that makes sense and reflect what it is!

**Review by micalexis — CHANGES_REQUESTED** [review]

File: kjourney/src/app/main/main.component.html (hunk 2 above)
Comment (jessicamcmillen) [reply kjourney/src/app/main/main.component.html:47]: Doesn't over ride css when in file, has to be in line as this is off materialise

**Review by micalexis — APPROVED** [review]

**Pull request merged**